lighthouse-logger/index.js

```diff
@@ -104,6 +106,16 @@ class Log {
     Log._logToStdErr(`${prefix}:${level || ''}`, [method, snippet]);
   }
 
+  static time({msg, id, args=[]}, level='log') {
+    marky.mark(id);
+    Log[level]('status', msg, ...args);
+  }
+
+  static timeEnd({msg, id, args=[]}, level='verbose') {
+    Log[level]('statusEnd', msg, ...args);
+    return marky.stop(id);
```

Comment:
is this going to be used? Should update `lighthouse-logger/index.d.ts` or drop the `return` here?

Reply:
done